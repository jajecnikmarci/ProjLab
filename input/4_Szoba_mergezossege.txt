starttest
add room room1 3
add room room2 3 poisonous
add door room1 room2 door1
add item room1 FFP2 ffp2
add item room1 Camembert camembert1
add person room1 Student studentCamembert
add person room1 Student studentFFP2
pickup studentCamembert
pickup studentFFP2
printstate studentCamembert
printstate studentFFP2
gotoroom studentCamembert room2
printstate room2
printstate studentCamembert
gotoroom studentFFP2 room2
printstate room2
printstate studentFFP2
endtest
starttest
add room room4 3
add item room4 Camembert camembert2
add room room5 3 poisonous
add door room4 room5 door2
add person room4 Professor professorCamembert
pickup professorCamembert
printstate professorCamembert
gotoroom professorCamembert room5
printstate room5
printstate professorCamembert
endtest
starttest
add room room6 3
add item room6 FFP2 ffp2
add room room7 3 poisonous
add door room6 room7 door3
add person room6 Professor professorFFP2
pickup professorFFP2
printstate professorFFP2
gotoroom professorFFP2 room7
printstate room7
printstate professorFFP2
endtest
starttest
add room room8 3
add room room9 3 poisonous
add door room8 room9 door4
add person room8 Cleaner cleaner1
printstate cleaner1
gotoroom cleaner1 room9
printstate room9
printstate cleaner1
endtest
