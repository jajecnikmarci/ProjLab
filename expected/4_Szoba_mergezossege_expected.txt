studentCamembert
	- location: room1
	- stunned: false
	- inventory: 
		camembert1
	- killImmunities: 
	- poisonImmunities: 
	- souls: 3
studentFFP2
	- location: room1
	- stunned: false
	- inventory: 
		ffp2
	- killImmunities: 
	- poisonImmunities: 
		PoisonImmunity
	- souls: 3
room2
	- capacity: 3
	- people: 
		studentCamembert
	- doors: 
		door1
	- items: 
		camembert1
	- poisonEffects: 
		poisonEffects
	- stunEffects: 
	- lastCleaning: null
studentCamembert
	- location: room2
	- stunned: true
	- inventory: 
	- killImmunities: 
	- poisonImmunities: 
	- souls: 3
room2
	- capacity: 3
	- people: 
		studentCamembert
		studentFFP2
	- doors: 
		door1
	- items: 
		camembert1
	- poisonEffects: 
		poisonEffects
	- stunEffects: 
	- lastCleaning: null
studentFFP2
	- location: room2
	- stunned: false
	- inventory: 
		ffp2
	- killImmunities: 
	- poisonImmunities: 
		PoisonImmunity
	- souls: 3
professorCamembert
	- location: room4
	- stunned: false
	- Inventory: 
		camembert2
	- poisonImmunities: 
room5
	- capacity: 3
	- people: 
		professorCamembert
	- doors: 
		door2
	- items: 
		camembert2
	- poisonEffects: 
		poisonEffects
	- stunEffects: 
	- lastCleaning: null
professorCamembert
	- location: room5
	- stunned: true
	- Inventory: 
	- poisonImmunities: 
professorFFP2
	- location: room6
	- stunned: false
	- Inventory: 
		ffp2
	- poisonImmunities: 
		poisonImmunity
room7
	- capacity: 3
	- people: 
		professorFFP2
	- doors: 
		door3
	- items: 
	- poisonEffects: 
		poisonEffects
	- stunEffects: 
	- lastCleaning: null
professorFFP2
	- location: room7
	- stunned: false
	- Inventory: 
		ffp2
	- poisonImmunities: 
		poisonImmunity
cleaner1
	- location: room8
room9
	- capacity: 3
	- people: 
		cleaner1
	- doors: 
		door4
	- items: 
	- poisonEffects: 
	- stunEffects: 
	- lastCleaning: StickinessEffect
cleaner1
	- location: room9
